src/resources/IPXInterfaces/IPXInterface.model.ts

```diff
@@ -0,0 +1,95 @@
+import {
+    HostedInterfaceCondition,
+    HostedInterfaceConfiguration,
+    HostedInterfaceFacet,
+    HostedInterfaceResultTemplate,
+    HostedInterfaceResultTemplateBadge,
+    HostedInterfaceResultTemplateDetail,
+    HostedInterfaceTab,
+} from '../HostedInterfacesCore';
+
+export enum IPXConditionOperator {
+    isDefined = 'isDefined',
+    isNotDefined = 'isNotDefined',
+    MustMatch = 'mustMatch',
+    MustNotMatch = 'mustNotMatch',
```

Comment:
Your enum keys capitalization is not consistent.

Reply:
word. It's also inconsistent in the enum the insight panel uses which is exactly the same as this one.

@louis-bompart is there a policy or rule of thumb contributors should abide by when assessing when breaking changes are justified? For example here it looks like allowing ourselves to break things could set us up for future success as the "interface builder" feature takes shape.

Reply:
AFAIK these interfaces are only used by the service LOB so the anticipated friction when upgrading would be on us.